back/test/integration/TestTechMapsApi.js

```diff
@@ -63,26 +62,331 @@ describe("API", () => {
       expect(res).to.have.status(404);
     });
 
-    it.skip("should bump up version when modified techmap put", async() => {
+    it("should bump up version when modified techmap put", async () => {
       // todo: make sure all fields are checked.
+      const id = newTechMapId();
+
+      const newTechMap = {
+        id,
+        name: "Хліб Французький (КХ)",
+        units: [1, 6],
+        steps: [
+          {
+            id: "STP-XXX-YYY-ZZZ-YY0",
+            name: "Замішування",
+            ingredients: [
+              {
+                ingredientId: "ING-XXX-YYY-ZZZ-YY0",
+                countByUnits: new Map([[1, 292], [6, 1752]])
+              }
+            ],
+            humanResources: [
+              {
+                peopleCount: 1,
+                countByUnits: new Map([[1, 15], [6, 22]])
+              }
+            ],
+            inventory: [
+              {
+                deviceId: "INV-XXX-YYY-ZZZ,YY0",
+                countByUnits: new Map([[1, 1], [6, 1]])
+              }
+            ],
+            instructions: `{"blocks":[{"key":"2ic2d","text":"Відважити воду та пшеничну закваску в чисту і суху ємність відповідного об'єму.","type":"ordered-list-item","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}},{"key":"7pdcs","text":"Окремо відважити боршно в чисту і суху ємність відповідного об’єму.","type":"ordered-list-item","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}},{"key":"an4s6","text":"В спіральному тістомісі змішати компоненти з послідовності (1) на протязі 4—5 хвилин (в залежності від кількості замісу).","type":"ordered-list-item","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}}],"entityMap":{}}`
+          }
+        ]
+      };
+
+      const insertRes = await chai
+        .request(app.server())
+        .post(`/techMaps/`)
+        .type("application/json")
+        .send(JSON.stringify(newTechMap));
+
+      expect(insertRes).to.have.status(201);
+
+      const getNewRes = await chai
```

Comment:
const insertRes = await chai
        .request(app.server())
        .post(`/techMaps/`)
        .type("application/json")
        .send(JSON.stringify(newTechMap));

      expect(insertRes).to.have.status(201);


We return 201 when we create new entity. In this case we rather modify so 200 is ok, unless we want to know which version we just inserted.

Reply:
Yeah, it could be useful for say implementing pessimistic redux-offline approach. To update state on "commit" action. Also needs discussion.